frontend/rolecall/src/app/cast/cast-drag-and-drop.component.scss

```diff
@@ -232,3 +235,114 @@ $cellHeight: 4rem;
   justify-content: space-between;
   width: 100%;
 }
+
+.button-h-wrapper {
```

Comment:
These classes are not used yet, let's leave them out and put them in later in the same PR where they are used, so that we don't have any unused code committed to 'master' branch at any point in time.

Reply:
I get it now. You meant to remove the below lines. I thought you were talking about the whole file.